task_process_inzendingen_voor_toezicht.py

```diff
@@ -105,3 +114,26 @@ def parse_inzending_sparql_response(inzending_res):
         log("Invalid value \"{}\" for boekjaar will be ignored, expected an int.".format(value))
 
     return inzending
+
+def exclude_inzendingen_from_rules(inzendingen)
+    """
+    This takes the bestuurseenheid (sender) and running ASK queries to check if it matches the pattern from business rules; see: Leesrechtenlogica Databank Erediensten,
+    then it will exclude the matching ones and construct the filtered list. 
+    """
+    filtered_inzendingen = []
+
+    for inzending in inzendingen:
+    
+        sender = inzending['bestuurseenheid']['value']
+
+        eb_has_cb = query(verify_eb_has_cb_exclusion_rule(sender))['boolean']
```

Comment:
That's in the good direction, but I think:
1. from an high level overview it's to strict. Your function signature seems to suggest it only filters on bestuurseenheid, but diving deeper into the implementation you take bestuurseenheid type and type of inzending. This is confusing at best.
2. per inzending you will query a vast subset of inzendingen of a bestuurseenheid. This is way too demanding for the database, since there are sometimes a lot.
What's wrong with `(verify_cb_exclusion_rule(inzending)`?

Reply:
1. Check, will improve that.
2. That was the main reason why I opted for altering the query with one call rather than manipulating the result of the query with multiple small calls, I'll have to rethink on how it can be more efficient but I'm not sure how to proceed yet.